java/core/src/main/java/com/whylogs/core/DatasetProfile.java

```diff
@@ -94,9 +99,10 @@ public void setDatasetTimestamp(Instant datasetTimestamp) {
 
   private void initializeNewColumns(Set<String> colNames) {
     for (String column : colNames) {
-      ColumnSchema columnSchema = this.schema.columns.get(column);
+      ColumnSchema columnSchema = this.schema.getColumns().get(column);
```

Comment:
why not use` this.schema.get(column)` here, the `getColumns()` is unnecessary, then restructure by using the returned optional.

Reply:
Old habit on what was considered better form. I'm happy to just do the get.